test/cri-containerd/jobcontainer_test.go

```diff
@@ -131,6 +140,85 @@ func Test_RunContainer_Hostname_JobContainer_WCOW(t *testing.T) {
 	}
 }
 
+func makeLocalGroup(name string) error {
+	output, err := exec.Command("net", "localgroup", name, "/add").CombinedOutput()
+	if err != nil {
+		return fmt.Errorf("failed to create localgroup %s with %s, output %s", name, err, string(output))
+	}
+	return nil
+}
+
+func deleteLocalGroup(name string) error {
+	output, err := exec.Command("net", "localgroup", name, "/delete").CombinedOutput()
+	if err != nil {
+		return fmt.Errorf("failed to delete localgroup %s with %s, output %s", name, err, string(output))
+	}
+	return nil
+}
+
+// Checks if userName is present in the group `groupName`
+func checkLocalGroupMember(groupName, userName string) error {
+	output, err := exec.Command("net", "localgroup", groupName).CombinedOutput()
+	if err != nil {
+		return fmt.Errorf("failed to check members for localgroup %s with %s, output %s", groupName, err, string(output))
+	}
+	if !strings.Contains(string(output), userName) {
+		return fmt.Errorf("user %s not present in the local group %s", userName, groupName)
+	}
+	return nil
+}
```

Comment:
Should we be instead using the winapi calls you made?

Reply:
Making a group required a bunch more bindings for code that was just used here for these tests so I shied off. This is simple enough

Reply:
Oh I see. In that case, this is fine.